scout/server/blueprints/variant/templates/variant/variant_details.html

```diff
@@ -240,19 +254,23 @@
                      </span>
                   {% endif %}
                   {% endif %}
-                  {% for data in obs.cases %}
-                     {% if data.variant and data.variant.category == "snv"%}
-                       <a class="badge rounded-pill bg-light text-dark" target="_blank" href="{{ url_for('variant.variant', institute_id=data.case.owner, case_name=data.case.display_name, variant_id=data.variant._id) }}">{{ data.case.display_name }}</a>
-                     {% elif data.variant and data.variant.category == "sv"%}
-                       <a class="badge rounded-pill bg-light text-dark" target="_blank" href="{{ url_for('variant.sv_variant', institute_id=data.case.owner, case_name=data.case.display_name, variant_id=data.variant._id) }}">{{ data.case.display_name }}</a>
-                     {% else %}
-                       <span data-bs-toggle="tooltip" title="Missing link, this might be caused by variants not loaded after a rerun or inexact SV matching.">
-                         <span class="ml-3 badge rounded-pill bg-light text-dark">{{ data.case.display_name }}</span>
-                       </span>
-                     {% endif %}
+                  {% for observed_case in obs.cases %}
+                    {% if loop.index <= 10 %}
```

Comment:
Nice update! just for the sake of testing with real data, do you mind if I lower this number to 1 or 2. It should work on our stage server

Reply:
Of course, go ahead!